tracim/lib/utils/request.py

```diff
@@ -214,6 +214,8 @@ def _get_current_comment(
         comment_id = ''
         try:
             if 'comment_id' in request.matchdict:
+                if not request.matchdict['comment_id'].isdecimal():
+                    raise ContentNotFoundInTracimRequest('comment_id is not a correct integer')  # nopep8
```

Comment:
isint() is not a valid python method as far i know, i choose isdecimal() method from std lib of string because it the more strict method i found to select this kind of id.
Not sure what i should do.

Reply:
This should be checked in #93, not here.